### PR TITLE
ENG-24 Added link Static Pages in left menu

### PR DESCRIPTION
**Pull request by E-zh**
Added link Static Pages in left menu

### CODE REVIEW

File: app/Http/ViewComposers/Admin/LeftMenuComposer.php [hunk 1]
```diff
@@ -16,6 +16,13 @@ public function compose(View $view)
                 'icon' => 'fa fa-th',
                 'active' => 'index-admin',
             ],
+
+            [
```
Comment (garf) [inline app/Http/ViewComposers/Admin/LeftMenuComposer.php]: This is hardcoded logic and not checks if static-pages package is installed.
So you have to create some private method that iterates package-names and if they are installed, adds them to menu.

File: app/Http/ViewComposers/Admin/LeftMenuComposer.php [hunk 2]
```diff
@@ -16,8 +16,26 @@ public function compose(View $view)
                 'icon' => 'fa fa-th',
                 'active' => 'index-admin',
             ],
+
```
Comment (garf) [inline app/Http/ViewComposers/Admin/LeftMenuComposer.php]: please remove

File: app/Http/ViewComposers/Admin/LeftMenuComposer.php [hunk 3]
```diff
@@ -16,8 +16,26 @@ public function compose(View $view)
...
         ];
 
+        $packages = ['static-pages'];
+
+        foreach ($packages as $package) {
+
+            if (config()->has($package)) {
```
Comment (garf) [inline app/Http/ViewComposers/Admin/LeftMenuComposer.php]: Will the config have this?

Comment (garf) [inline app/Http/ViewComposers/Admin/LeftMenuComposer.php]: Question is still actual :)

Comment (E-zh) [inline app/Http/ViewComposers/Admin/LeftMenuComposer.php]: It works

**Pull request closed without merging**